opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp

```diff
@@ -188,20 +184,16 @@ BlackoilAquiferModel<TypeTag>::init()
         const Aquifetp aquifetp = Aquifetp(deck);
         const Aquancon aquifer_connect = Aquancon(eclState.getInputGrid(), deck);
 
-        std::vector<Aquifetp::AQUFETP_data> aquifersData = aquifetp.data();
-        std::vector<Aquancon::AquanconOutput> aquifer_connection = aquifer_connect.getAquOutput();
-
         assert(aquifersData.size() == aquifer_connection.size());
```

Comment:
Same here. `aquifersData` is not defined.

Reply:
fixed